examples/v4/category-dataview.html

```diff
@@ -41,16 +41,10 @@
       L.tileLayer('https://cartodb-basemaps-{s}.global.ssl.fastly.net/rastertiles/voyager_nolabels/{z}/{x}/{y}.png').addTo(map);
 
       // Create the Client
-      // var client = new carto.Client({
-      //   apiKey: '84fdbd587e4a942510270a48e843b4c1baa11e18',
-      //   serverUrl: 'https://{user}.carto.com:443',
-      //   username: 'cartojs-test'
-      // });
-
       var client = new carto.Client({
-        apiKey: '4bf183ec5867bd5f9eb59d4d33693fa4fb4283bf',
-        serverUrl: 'http://{user}.localhost.lan:8181',
-        username: 'cdb'
+        apiKey: '84fdbd587e4a942510270a48e843b4c1baa11e18',
+        serverUrl: 'https://{user}.carto.com:443',
```

Comment:
This won't work soon. Could you please replace it by `'https://cartojs-test.carto.com:443'`?

Reply:
:)